### ISSUE TITLE
Typo in axe extension for Rule ID: list

### ISSUE THREAD

**Issue by KB55** [dequelabs/axe-core#3559]
Product: axe Extension
Expectation: Correctly describe how to fix the issue
Actual: Typo in the text 
Motivation: The typo has caused confusion and caused developers to look in the wrong place for the error

Extension version: v4.32.0
axe-core version: v4.4.2
Chrome browser version 103.0.5060.134 (Official Build) (64-bit)

In the axe extension an error has been identified for:

> Rule ID: list 
> Ruleset: axe-core 4.4
> `<ul>` and `<ol>` must only directly contain `<li>`, `<script>` or `<template>` elements

The issue was triggered because a `<span>` was nested within a `<ul>`. The extension says: 

> Fix the following: List element has direct children that are not allowed inside `<li>` elements

I believe this is a typo and that it should say:
> List element has direct children that are not allowed inside `<ul>` elements

The rule is concerned with errors due to incorrect direct children of `<ul>` and `<ol>` elements, not `<li>` elements and therefore this fix shown in the extension is misleading. 

Screenshots attached:

- Axe issue
- Axe issue description
- Code for issue

![AxeExtensionScreenshot](https://user-images.githubusercontent.com/26979111/180433775-90b6c897-b9ed-49c2-bbc8-11a4e372c013.png)
![AxeIssueDescScreenshot](https://user-images.githubusercontent.com/26979111/180433803-9de06360-e0e9-415e-9efb-a2d455c2423b.png)
![AxeCodeExample](https://user-images.githubusercontent.com/26979111/180433814-b5eb829b-1fe6-483a-a608-f6244c3303a6.png)

**Comment by straker** [comment]
Thanks for the issue. I can see where the confusion is. I'll talk to @WilcoFiers about it and we'll see what we can do.

**Comment by WilcoFiers** [comment]
Agreed, this is wrong. `only-listitems` and `only-dlitems` need to be updated. Thank you for raising this.

**Comment by padmavemulapati** [comment]
@straker , I am still seeing the fix description as 
" List element has direct children that are not allowed inside <li> elements" 
for the script:
```
<ul id="scriptproperul">
    <script></script>
    <template></template>
    <li>One.</li>
    <li>Two.</li>
  </ul>
  <ul class="sample"></ul>
  <ol id="emptyol"></ol>
  <ol id="scriptemptyol">
```
<img width="819" alt="image" src="https://user-images.githubusercontent.com/47409579/196385647-3b3874c3-7148-4f92-99aa-ea7623def7f6.png">
